pkgs/tools/inputmethods/input-remapper/default.nix

```diff
@@ -35,13 +35,13 @@ let
 in
 (buildPythonApplication {
   pname = "input-remapper";
-  version = "1.5.0";
+  version = "2.0.0";
 
   src = fetchFromGitHub {
-    rev = "e31a1b2bc5d23fe13130afcc242063196335399f";
+    rev = "9e04df79ca4dca77516a9864d740b9b2d4aef788";
     owner = "sezanzeb";
     repo = "input-remapper";
```

Comment:
```suggestion
    owner = "sezanzeb";
    repo = "input-remapper";
    rev = version;
```

Reply:
Not great for the use of final.src.rev on 155

Reply:
IIRC if there are tags those should be used. We could add the commit into an extra variable or leave it out and replace it with a fake hash.

Reply:
Using the rev now